adoc/bp_chap_expanded_support.adoc

```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
 // Reason: RMT and SMT cannot mirror Debian repositories (yet) and so cannot create it in that directory the correct files.
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
+== Prepare {ubuntu} Clients for Registration
 
-This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
+Some preparation is required before you can register {ubuntu} clients to {productname} Server.
 
+Before you begin, ensure you have the {ubuntu} product enabled, and have synchronized the {ubuntu} channels:
 
-.Procedure: Preparing an {Ubuntu} 16.04 Client for Registration
+For {ubuntu} 18.04:
 
-. On the client, open the [filename]``/etc/apt/sources.list.d/suma_client_tools.list`` file, and add this line:
-+
-----
-deb http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu16.04-SUSE-Manager-Tools/xUbuntu_16.04/ /
-----
-+
-. Edit the [filename]``sudoers`` file:
+* Product: {ubuntu} Client 18.04
+* Mandatory channels: [systemitem]``ubuntu-18.04-pool-amd64``
+
+For {ubuntu} 16.04:
+
+* Product: {ubuntu} Client 16.04
+* Mandatory channels: [systemitem]``ubuntu-16.04-pool-amd64``
+
+[NOTE]
+====
+The mandatory channels do not contain {ubuntu} upstream packages.
+The repositories and channels for synchronizing upstream content must be configured manually.
+====
+
+
+
+.Procedure: Preparing an {ubuntu} Client for Registration
+
+. Ensure that you have the appropriate software channels available on your system.
+In the {productname} {webui}, navigate to menu:Software[Channel List  > All].
+You should see a base channel and a child channel for your architecture, for example:
 +
 ----
-sudo visudo
+ ubuntu-18.04-pool for amd64
+ |
+ +- Ubuntu-18.04-SUSE-Manager-Tools for amd64
 ----
+. Create custom repositories to mirror the {ubuntu} packages.
+For more information on creating custom repositories, see xref:modules/administration/pages/channel-management.adoc[].
+For example:
 +
-Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will bootstrap the client in the {webui}:
+For `main`:
+
+* Repository Label: ubuntu-bionic-main
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/
+* Repository Type: deb
 +
-----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
-----
+For `main-updates`:
```

Comment:
Seems the indentation is not correct.

Reply:
This is showing up correctly in the source. I think the diff  viewer is doing something strange.

Reply:
No, please check when you click on "View File". There the indentation is wrong.